MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,21 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func navigationServiceDidChangeAuthorization(_ service: NavigationService, _ manager: CLLocationManager) {
+        // CLLocationManager.accuracyAuthorization was introduced in the iOS 14 SDK in Xcode 12, so Xcode 11 doesn’t recognize it.
+        guard let accuracyAuthorizationValue = manager.value(forKey: "accuracyAuthorization") as? Int
+        else { return }
+        let accuracyAuthorization = MBNavigationAccuracyAuthorization(rawValue: accuracyAuthorizationValue)
+        
+        if #available(iOS 14.0, *), accuracyAuthorization == .reducedAccuracy {
+            let title = NSLocalizedString("ENABLE_PRECISE_LOCATION", bundle: .mapboxNavigation, value: "Enable precise location to navigate", comment: "Label indicating precise location is off and needs to be turned on to navigate")
+            showStatus(title: title, spinner: false, duration: 20, animated: true, interactive: false)
```

Comment:
Is there a way to make this status stay up until precise location is enabled, or is that the focus of another issue?

Reply:
Before merging, please open an issue to track persisting this banner until precise location is reenabled, so that we don’t lose track of that important detail. #2691 is sort of related, but I think persisting the accuracy authorization banner and automatically hiding it at the right time would be more tractable, because we’re already observing the notification that we need to observe to know when to hide the banner.

Reply:
#2723